src/orb.c

```diff
@@ -1534,7 +1534,11 @@ static Int InitLibrary ( StructInitInfo *module )
     /* init filters and functions */
     InitGVarFuncsFromTable(GVarFuncs);
 
-    tmp = NEW_PREC(0);
+    tmp = NEW_PREC(1);
+    Obj p = NEW_PERM2(0);
+    AssPRec(tmp, RNamName("PERM_HASH_SKIP"),
+            INTOBJ_INT((UInt) ADDR_PERM2(p) - (UInt) ADDR_OBJ(p)));
+    CHANGED_BAG(tmp);
```

Comment:
This looks fine, except it is missing a comment: in years, somebody will stumble over this code and wonder what it's a bout. So perhaps add something like: `// In GAP 4.9, the memory layout of T_PERM objects changed, the following code helps with that`  -- I think that's enough, people can go on from there.